src/components/panel/AccountPanel.vue

```diff
@@ -1,187 +1,95 @@
 <template>
-  <BaseCard>
-    <template #header>
-      <AccountHeader :has-all-shops-linked="hasAllShopsLinked">
-        {{ $tc('psaccounts.account.title', shopsWithUrl.length) }}
-      </AccountHeader>
-    </template>
-    <template #body>
-      <ShopUrlShouldExistsAlert
-        v-if="hasShopsWithoutUrl"
-        :has-all-shops-without-url="hasAllShopsWithoutUrl"
-        :shop-names-without-url="shopNamesWithoutUrl"
+  <puik-card
+    class="acc-p-6"
+  >
+    <div class="acc-flex acc-flex-row acc-items-center">
+      <PuikIcon
+        v-if="hasShopsLinked"
+        class="acc-text-white acc-bg-green-500 acc-rounded-full acc-p-1 acc-mr-2"
+        icon="check"
+        data-testid="account-panel-linked-icon"
       />
-
-      <div
-        class="acc-flex acc-flex-col acc-items-center md:acc-flex-row"
-        :class="{'acc-mt-2': hasShopsWithoutUrl}">
-        <AccountShopLinkMessage
-          class="md:acc-mr-2"
-          :shops="shopsWithUrl" />
-        <AccountLinkToUi
-          class="acc-mt-2 md:acc-mt-0"
-          :accounts-ui-url="accountsUiUrl"
-          :app="app"
-          :backend-user="backendUser"
-          :onboarding-link="onboardingLink"
-          :shops="shopsWithUrl"
-          :shop-context="shopContext"
-        />
-      </div>
-
-      <ModuleUpdateInformationAlert
-        v-if="isLinkedV4"
-        class="acc-mt-6"
-      />
-
-      <UserEmailNotValidatedAlert
-        v-if="userHasEmailNotVerified"
-        class="acc-mt-6"
-        :sso-resend-verification-email="ssoResendVerificationEmail"
+      <p class="acc-m-0 puik-h5">
+        {{ t('psaccounts.account.title', shopsInContext.length) }}
+      </p>
+    </div>
+    <div
+      class="acc-flex acc-flex-col acc-items-center md:acc-flex-row"
+    >
+      <AccountShopLinkMessage
+        class="md:acc-mr-2"
+        :shops-in-context="shopsInContext"
+        :shop-context="shopContext"
       />
-
-      <UserNotSuperAdminAlert
-        v-if="!backendUser.isSuperAdmin"
-        class="acc-mt-6"
-        :super-admin-email="superAdminEmail"
+      <AccountLinkToUi
+        v-if="!shopsWithoutUrl.length"
```

Comment:
I don't know if it's a regression or a desired behavior but when I have one shop without URL, I don't have the link / manage linked stores buttons in his group context or all context

Reply:
It's a desired behavior, when there is at least one shop without an url the user is blocked and must fix that in order to proceed

Reply:
Ok ! It's strange because I think some owner of shop keep a shop for history or other case and just disable the URL to make the shop inaccessible from users

Reply:
It seems that you can't disable a shop in multishop, you can delete it tho
Same for the url it can't be empty once you have set an url